locales/en/package.i18n.json

```diff
@@ -1,21 +1,22 @@
 {
-    "deviceSimulatorExpressExtension.commands.common.installDependencies": "Install Extension Dependencies",
-    "deviceSimulatorExpressExtension.commands.common.label": "Device Simulator Express",
-    "deviceSimulatorExpressExtension.commands.common.runSimulator": "Run Simulator",
-    "deviceSimulatorExpressExtension.commands.common.gettingStarted": "Getting Started",
-
-    "deviceSimulatorExpressExtension.commands.common.changeBaudRate": "Change Baud Rate",
-    "deviceSimulatorExpressExtension.commands.common.closeSerialMonitor": "Close Serial Monitor",
-    "deviceSimulatorExpressExtension.commands.common.openSerialMonitor": "Open Serial Monitor",
-    "deviceSimulatorExpressExtension.commands.common.selectSerialPort": "Select Serial Port",
-    "deviceSimulatorExpressExtension.commands.cpx.openSimulator": "[Circuit Playground Express] Open Simulator",
-    "deviceSimulatorExpressExtension.commands.cpx.newFile": "[Circuit Playground Express] New File",
-    "deviceSimulatorExpressExtension.commands.cpx.deployToDevice": "[Circuit Playground Express] Deploy to Device",
-    "deviceSimulatorExpressExtension.commands.microbit.deployToDevice": "[micro:bit] Deploy to Device",
-    "deviceSimulatorExpressExtension.commands.microbit.openSimulator": "[micro:bit] Open Simulator",
-    "deviceSimulatorExpressExtension.commands.microbit.newFile": "[micro:bit] New File",
-    "deviceSimulatorExpressExtension.configuration.title": "Device Simulator Express configuration",
-    "deviceSimulatorExpressExtension.configuration.properties.configEnvOnChange": "When you change the Python interpreter, the Device Simulator Express will automatically configure itself for the required dependencies.",
-    "deviceSimulatorExpressExtension.configuration.properties.debuggerPort": "The port the Server will listen on for communication with the debugger.",
-    "deviceSimulatorExpressExtension.configuration.properties.dependencyChecker": "Whether or not to ask if we can download dependencies. If unchecked, the extension will default to never download dependencies, except when automatically creating a virtual environment in the extension files."
-}
+  "deviceSimulatorExpressExtension.commands.common.installDependencies": "Install Extension Dependencies",
```

Comment:
Can we not include this because it has no changes ;~; ?

Reply:
I'm not sure why, everytime there's a change here, it changes the whole file. There were added commands for clue

Reply:
changed the spacing, will commit soon